ui/src/app/eft/eft.component.css

```diff
@@ -0,0 +1,6 @@
+.eftLabels{
+  text-align: right;
+}
+.eftPaymentOptions{
+  text-align: center;
+}
```

Comment:
You could have saved quite a bit of duplicate code by using the provided alignment utilities http://getbootstrap.com/docs/4.1/utilities/text/#text-alignment

Reply:
sure.